pilot/pkg/status/state.go

```diff
@@ -170,8 +172,9 @@ func (c *DistributionController) writeStatus(ctx context.Context, config Resourc
 	// Note: I'd like to use Pilot's ConfigStore here to avoid duplicate reads and writes, but
 	// the update() function is not implemented, and the Get() function returns the resource
 	// in a different format than is needed for k8s.updateStatus.
-	c.currentlyWriting.Lock(config)
-	defer c.currentlyWriting.Unlock(config)
+	//c.currentlyWriting.Mock(config)
+	//defer c.currentlyWriting.Unlock(config)
+	log.Warnf("writing status for config %v", config)
```

Comment:
This shouldn't be a warning - probably shouldn't even be logged as it will be very spammy?